src/components/SelectRegisteredAsset.vue

```diff
@@ -166,26 +152,23 @@ export default class SelectRegisteredAsset extends Mixins(TranslationMixin, Dial
     return this.isSoraToEthereum ? 'address' : 'externalAddress'
   }
 
+  get balanceSymbol (): string {
+    return this.isSoraToEthereum ? 'balance' : 'externalBalance'
```

Comment:
Could you move this to utils please? It seems, that we already have 4 places with this check.